in_each_e2e.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset
+
+# Run arguments passed in each e2e folder in serial
+for dir in e2e/*; do
```

Comment:
I've seen you write this code so many different ways now 😆

Reply:
🤷